### PR TITLE
docs: readme standard updates

### PR DESCRIPTION
**Pull request by thephez**
## Issue being fixed or feature implemented

Brings readme into better [standard-readme](https://github.com/RichardLitt/standard-readme) compliance. Also aligns with changes previously made on master for 0.13.

## What was done?

- [x] Update ToC and make minor formatting/wording adjustments
- [x] Update code example
- [x] Update badges

## How Has This Been Tested?

N/A

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,29 +35,37 @@ npm install @dashevo/dapi-client
 const DAPIClient = require('@dashevo/dapi-client');
 var client = new DAPIClient();
```
Comment (shumkov) [inline README.md:36]:
```suggestion
const client = new DAPIClient();

const coreStatus = await client.core.getStatus();

console.dir(coreStatus);
```

File: README.md [hunk 2]
```diff
@@ -36,29 +35,37 @@ npm install @dashevo/dapi-client
...
 
-Custom seed nodes are necessary for connecting the client to devnets since the
-client library is unaware of them otherwise.
+Custom seed nodes are necessary for connecting the client to devnets since the client library is unaware of them otherwise.
 
 ```javascript
 const DAPIClient = require('@dashevo/dapi-client');
 
 var client = new DAPIClient({
   seeds: [{
```
Comment (shumkov) [inline README.md:51]:
```suggestion
  seeds: [{
     host: 'seed-1.evonet.networks.dash.org',
     httpPort: 3000,
     grpcPort: 3010,
  }]
```

File: README.md [hunk 3]
```diff
@@ -36,29 +35,37 @@ npm install @dashevo/dapi-client
...
     port: 3000
   }],
 });
 
-var blockHeight = client.getBestBlockHeight();
+client.getBestBlockHash().then((r) => {
+  console.log(r);
+});
+
 ```
```
Comment (shumkov) [inline README.md:62]: I would add one more section on how to specify specific node to make request (without DML) with addresses option.

Also, I think we should mention that now you can pass all DAPI Client options directly to any command to have specific behavior for that call.

Comment (thephez) [reply README.md:67]: Why don't we merge these changes and a separate PR for additional updates?

Comment (shumkov) [reply README.md:67]: Sounds good!

**Pull request merged**